frontend/src/static/js/v_authorship.js

```diff
@@ -59,6 +61,11 @@ window.vAuthorship = {
         this.selectedFileTypes = this.fileTypes.slice();
       }
     },
+
+    info() {
+      this.initiate();
+      this.setInfoHash();
```

Comment:
this seems like a duplication of `created()`, is it necessary to have both?

Reply:
Yes I think so.

Previously, when authorship tab is already open, clicking to another author's code will close the tab and open the tab to trigger the redrawing of the entire authorship tab, which will start over and call `created()` of the authorship tab.

This PR removes that unnecessary redrawing logic, so `created()` won't be called again when user clicks on another author's code. That is why we need a watcher that listens to the changes in `info`, and updates the authorship tab accordingly.